converter/SEMSoutput.py

```diff
@@ -213,14 +213,20 @@ def add_entry(precinct_id, contest_id, answer):
             if answers != None:
                 num_seats = contest["seats"] if contest['type'] == 'candidate' else 1
 
-                # blank vote
-                if len(answers) == 0:
-                    add_entry(precinct_id, contest["id"], BLANKVOTE_CANDIDATE["id"])
-                    continue
+                # under votes
+                if len(answers) < num_seats:
```

Comment:
pedantic nit: technically you don't need this `if`, since `range(5,5)` will give you an empty list or the next loop. (I'm totally fine if you prefer to ignore this comment, though.)